sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/Configs.java

```diff
@@ -175,6 +174,19 @@ public class Configs {
     public static final String DIAGNOSTICS_PROVIDER_SYSTEM_EXIT_ON_ERROR = "COSMOS.DIAGNOSTICS_PROVIDER_SYSTEM_EXIT_ON_ERROR";
     public static final boolean DEFAULT_DIAGNOSTICS_PROVIDER_SYSTEM_EXIT_ON_ERROR = true;
 
+    // Metrics
+    // Samples:
+    //            System.setProperty(
+    //                "COSMOS.METRICS_CONFIG",
+    //                "{\"metricCategories\":\"[OperationSummary, RequestSummary]\","
+    //                + "\"tagNames\":\"[Container, Operation]\","
+    //                + "\"sampleRate\":0.5,"
+    //                + "\"percentiles\":[0.90,0.99],"
+    //                + "\"enableHistograms\":false,"
+    //                + "\"applyDiagnosticThresholdsForTransportLevelMeters\":true}");
+    public static final String METRICS_CONFIG = "COSMOS.METRICS_CONFIG";
+    public static final String DEFAULT_METRICS_CONFIG = new CosmosMicrometerMetricsConfig().toJson();
```

Comment:
If its a new instance of `CosmosMicrometerMetricsConfig()` class with default variables and their declared values converted to a string, why do we need the instance at first place? Can't we use something static string to represent the same new instance as a string without creating extra memory for the instance?